source/server/active_tcp_socket.cc

```diff
@@ -89,7 +111,14 @@ void ActiveTcpSocket::continueFilterChain(bool success) {
           no_error = false;
           break;
         } else {
-          // Blocking at the filter but no error
+          // When reach this branch, the maxReadBytes must greater than 0.
```

Comment:
Why? Currently, this comment doesn't tell me anything more than the ASSERT. Maybe comment why, or delete the comment.

Reply:
The reason is same with what you said here https://github.com/envoyproxy/envoy/pull/17395/files/ec80e8825245aa0ed651ec63c587aa77e3d53c40#r825086207

I will add comment here.